python/ray/_private/runtime_env/nsight.py

```diff
@@ -0,0 +1,132 @@
+import os
+import logging
+import subprocess
+import copy
+from typing import Tuple, List, Dict, Optional
+
+from ray._private.runtime_env.context import RuntimeEnvContext
+from ray._private.runtime_env.plugin import RuntimeEnvPlugin
+from ray._private.utils import (
+    try_to_create_directory,
+)
+
+default_logger = logging.getLogger(__name__)
+
+NSIGHT_DEFEAULT_CONFIG = {
+    "-t": "cuda,cudnn,cublas,nvtx",
+    "-o": "'worker_process_%p'",
+    "--cudabacktrace": "true",
```

Comment:
According to the docs, supported values here are "all, none, kernel, memory, sync, other"